website/src/theme/DocPage/index.js

```diff
@@ -32,8 +32,9 @@ function DocPageContent({ currentDocRoute, versionMetadata, children }) {
     apisix : "getting-started",
     apisixDashboard : "dashboard",
     apisixIngressController : "what-is-apisix-ingress-controller",
-    apisixHelmChart : "install",
-    apisixDocker : "build-an-image-from-source"
+    apisixHelmChart : "seeking-help",
+    apisixDocker : "build-an-image-from-source",
+    apisixJavaPluginRunner : "overview"
```

Comment:
Hi @iamayushdas I'm not so familiar with this. What does this `overview` corresponding to?

Reply:
Its the id of page

Reply:
Like this one?
![image](https://user-images.githubusercontent.com/34589752/123681146-7002a100-d817-11eb-96a1-a0992d3b8e6d.png)

But it seems for other projects the list here is not corresponding to the link it jumps to like
![image](https://user-images.githubusercontent.com/34589752/123681242-958faa80-d817-11eb-9ee9-e5e7f93d05fe.png)

Reply:
I know it will be confusing for many developers, thats why i wanted to add new features for uniform id's but spacewander was not happy with the idea, 

You could visit this for your query - any please put your values opinion too

Reply:
Got it